cs_api_cli/cs_api_cli.ml

```diff
@@ -146,6 +147,12 @@ let project_id =
   in
   Cmdliner.Arg.(value & opt (some int) None & info ["p"; "project-id"] ~doc)
 
+let slot_name =
+  let doc =
+    "Name of the (existing or not) slot the trace should be added to."
```

Comment:
**Question:** a non-existing slot can have a name?

Reply:
If there's no slot with this name then one will be created.

Reply:
Can you add this explanation then? It's kind of confusing without it

Reply:
I modified the explanation. Does it look better this way to you?

Reply:
Perfect!